src/AspNet.Security.OAuth.FourSquare/FourSquareAuthenticationOptions.cs

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNet.Authentication.OAuth;
+using Microsoft.AspNet.Http;
+
+namespace AspNet.Security.OAuth.FourSquare {
+    /// <summary>
+    /// Defines a set of options used by <see cref="FourSquareAuthenticationHandler"/>.
+    /// </summary>
+    public class FourSquareAuthenticationOptions : OAuthOptions {
+        public FourSquareAuthenticationOptions() {
+            AuthenticationScheme = FourSquareAuthenticationDefaults.AuthenticationScheme;            
+            Caption = FourSquareAuthenticationDefaults.Caption;
+            ClaimsIssuer = FourSquareAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(FourSquareAuthenticationDefaults.CallbackPath);
+
+            AuthorizationEndpoint = FourSquareAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = FourSquareAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = FourSquareAuthenticationDefaults.UserInformationEndpoint;
+
+            SaveTokensAsClaims = false;
+
+            ApiVersion = FourSquareAuthenticationDefaults.ApiVersion;           
```

Comment:
You can remove this line and set the property inline:

`public string ApiVersion { get; set; } = FourSquareAuthenticationDefaults.ApiVersion`